### PR TITLE
feat(clerk-js,backend,types): Use new imageUrl field and enable new avatar animation

### PR DESCRIPTION
**Pull request by desiprisg**
## Type of change

- [ ] 🐛 Bug fix
- [x] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [x] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [x] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/localizations`
- [ ] `@clerk/clerk-expo`
- [x] `@clerk/backend`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/fastify`
- [ ] `@clerk/chrome-extension`
- [ ] `gatsby-plugin-clerk`
- [ ] `build/tooling/chore`

## Description

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.

### CODE REVIEW

File: packages/backend/src/api/resources/JSON.ts [hunk 1]
```diff
@@ -128,7 +128,11 @@ export interface OrganizationJSON extends ClerkResourceJSON {
   object: ObjectType.Organization;
   name: string;
   slug: string | null;
+  /**
+   * @deprecated Use image_url instead
+   */
   logo_url: string | null;
+  imageUrl: string;
```
Comment (anagstef) [inline packages/backend/src/api/resources/JSON.ts:135]: ❓ Should this be `image_url` ?

File: packages/types/src/user.ts [hunk 2]
```diff
@@ -47,7 +47,11 @@ export interface UserResource extends ClerkResource {
   fullName: string | null;
   firstName: string | null;
   lastName: string | null;
+  /**
+   * @deprecated Use image_url instead
```
Comment (anagstef) [inline packages/types/src/user.ts:51]:
```suggestion
   * @deprecated Use imageUrl instead
```

**Review by anagstef — APPROVED** [review]

File: packages/types/src/user.ts (hunk 2 above)
Comment (desiprisg) [reply packages/types/src/user.ts]: fixed

File: packages/backend/src/api/resources/JSON.ts (hunk 1 above)
Comment (desiprisg) [reply packages/backend/src/api/resources/JSON.ts]: yeap, fixed

**Pull request closed without merging**